src/doubleTrouble.js

```diff
@@ -25,6 +25,13 @@
  */
 function doubleTrouble(array, target) {
   // write code here
+  return array.reduce(
+    (resultArray, value) => {
```

Comment:
you have access to the array and can simplify `resultArray[resultArray.length - 1]`
```suggestion
    (resultArray, value, index, iteratedArray) => {
```

Reply:
Very good idea! thank You very much!

Reply:
Ouch! Stop! If I understood rightly, index is reffered to iterated array and means index of current element! that will be wrong code! Or I didn't understand what to do...

Reply:
P.S. I commit the suggestion. How to back?